### PR TITLE
Chore/new stake page

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Created view to onboard new nominators.

1. If there are no accounts created, then it will open a pop-up to let user create/import account before landing on this page. However, user can click on later and that would display error message on the page - no account exist
2. Bond and nominate will be executed as one extrinsic
3. Click on 'Know the risk' will open help dialog
4. If the bonded amount is greater than account's CENNZ balance - error message would be displayed - not enough to stake
5. Used styled components and added two colours in styled-theming/themes/dark/colors.ts and used all the colours from here.

### CODE REVIEW

File: packages/app-staking/src/md/basic.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 # Nominators
 
-Nominators have a responsibility to elect the most trustworthy validators to secure the network.  
-To become a nominator you must stake the minimum required CENNZ tokens and vote for your preferred validators.  
+Nominators have a responsibility to elect the most trustworthy validators to secure the network.
```
Comment (holygits) [inline packages/app-staking/src/md/basic.md:3]: @KarishmaBothara  the trailing whitespace is what markdown uses to create a newline

Comment (KarishmaBothara) [reply packages/app-staking/src/md/basic.md]: adding the trailing whitespace back.

**Comment by holygits** [comment]
@KarishmaBothara I'm in favour of merging this

**Review by holygits — APPROVED** [review]

**Comment by KarishmaBothara** [comment]
I have all the changes made in https://github.com/cennznet/ui/pull/90
which contains all the changes for New Stake and Manage stake + all the corrects (i.e rewards.payee used over staking.payee + inflation rate)

**Comment by KarishmaBothara** [comment]
Planning to close this for now..

**Pull request closed without merging**